tutorials/vr/openxr/index.rst

```diff
@@ -48,7 +48,7 @@ To turn this simple scene into a proper game, the sky is the limit.
 Below, there are a few more topics specific to this plugin.
 However, the following resources are a good place to continue:
 
-- `VR starter tutorial <https://docs.godotengine.org/en/stable/tutorials/vr/vr_starter_tutorial/vr_starter_tutorial_part_one.html>`__
+- `VR starter tutorial <https://docs.godotengine.org/en/stable/tutorials/vr/openvr/vr_starter_tutorial/vr_starter_tutorial_part_one.html>`__
```

Comment:
This should use an internal reference instead of an external link:

```suggestion
- :ref:`VR starter tutorial <doc_vr_starter_tutorial_part_one>`
```